File: Cs/Solution/FindClosest.cs
using System;

namespace Solution
{
	public partial class Solution
	{
		public int FindClosest(int x, int y, int z)
		{
			int diff = Math.Abs(x - z) - Math.Abs(y - z);
			if (diff < 0)
			{
				return 1;
			}
			else if (diff > 0)
			{
				return 2;
			}
			else
			{
				return 0;
			}

		}
	}
}

File: Ruby/solution/find_closest.rb
# @param {Integer} x
# @param {Integer} y
# @param {Integer} z
# @return {Integer}
def find_closest(x, y, z)
  d = (x - z).abs <=> (y - z).abs
  (d == 0) ? 0 : (d < 0 ? 1 : 2)
end
